tests/unit/fixtures/gameStates/resolveThree.js

```diff
@@ -47,7 +52,22 @@ export const resolveThree = {
       points: [ Card.ACE_OF_HEARTS ],
       faceCards: [],
     },
-    deck: [ Card.SIX_OF_SPADES, Card.TWO_OF_HEARTS, Card.ACE_OF_CLUBS ],
+    deck: [
+      Card.ACE_OF_CLUBS, Card.TWO_OF_CLUBS, Card.THREE_OF_CLUBS, Card.FOUR_OF_CLUBS, 
+      Card.FIVE_OF_CLUBS, Card.SIX_OF_CLUBS, Card.SEVEN_OF_CLUBS, Card.EIGHT_OF_CLUBS, 
+      Card.NINE_OF_CLUBS, Card.TEN_OF_CLUBS, Card.JACK_OF_CLUBS, Card.QUEEN_OF_CLUBS, 
+      Card.KING_OF_CLUBS,
```

Comment:
Just wondering: why are the kings on their own line? It's lonely at the top I suppose 😂

Reply:
🤣, I used chatGPT to help me get the unused cards... It took a few tries and didn't work out as well as I had hoped, it kept missing cards.

Reply:
Gotta be careful with any AI. In my experience it always missed something, changes something weird or does strange stuff the larger the context.

I try and keep it tightly defined in terms of rules and instructions and only send the bit I need changed lol.